DOCS-2988: Small fixes, descriptions, adjusted tags

FrancescaNCook opened the pull request:
Hey @Kris-MultiSafepay !
I started making some small capitalization adjustments, but then added some missing descriptions, and suggested a couple of new tags to break up the huge list under Accounts: namely Websites, UBOs, and Affiliated accounts. 
All that's really left under Accounts now is bank account related stuff, apart from that one random request about an authenticated user. Not sure what to do about that one yet...
See what you think and hit me up if you want to discuss anything!

Kris-MultiSafepay commented:
@FrancescaNCook, nice! I really like the idea of grouping with more intuitive and granular tags than just 'Account'. However, all requests under UBOs and Websites (except 'List website categories') can only be performed on affiliated accounts. So I would suggest grouping those under the 'Affiliated accounts' tag. 

Essentially that means splitting up 'Accounts' into just 'Accounts' and 'Affiliate accounts', which I still think is an improvement over a single huge group. Maybe we can find better groupings once all requests are documented.

FrancescaNCook commented:
@Kris-MultiSafepay Thanks for the feedback! I've adjusted the labels to Affiliate accounts, Affiliate websites, and Affiliate UBOs, because otherwise we end up with a massive list again under Affiliated accounts. I've moved List website categories back under Accounts. 
@Sarah-MultiSafepay Would you mind doing a quick linguistic review?

Sarah-MultiSafepay commented on openapi/paths/customers@{customer_id}.yaml:
```diff
@@ -1,6 +1,6 @@
 get:
-  description: Get customer by id
-  operationId: GetCustomer
+  description: Fetch information about a customer using their customer ID.
```
```suggestion
  description: Fetch information about a customer, using their customer ID.
```

Sarah-MultiSafepay commented on openapi/paths/customers@{customer_id}@delivery-details.yaml:
```diff
@@ -1,6 +1,6 @@
 get:
-  description: ''
-  operationId: ListDeliveryDetails
+  description: "Fetch a customer's address details."
```
Should we clarify this is a customer's **shipping** address?

FrancescaNCook replied:
@Sarah-MultiSafepay Yes, good suggestion. Thanks!

Sarah-MultiSafepay reviewed (commented):
@FrancescaNCook, reviewed! I only made two suggestions for your consideration.
